static/app/gettingStartedDocs/python/mongo.tsx

```diff
@@ -50,6 +50,15 @@ const onboarding: OnboardingConfig = {
       ),
       configurations: [
         {
+          description: params.isProfilingSelected
```

Comment:
I don't know why mongo is a platform since it's just an integration for the python mongo driver. This also never shows in the onboarding since performance/profiling is not added as an available product for the mongo platform. But I updated it for completeness.